.github/workflows/queries.yml

```
@@ -0,0 +1,139 @@
+      - '.github/workflows/queries.yml'
+      - 'sql/**'
+      - 'docqueries/**'
+      - 'tools/testers/**'
+      - 'CMakeLists.txt'
+      - 'tools/scripts/get_signatures.sh'
+      - 'tools/testers/doc_queries_generator.pl'
+
+    branches-ignore:
+      - 'gh-pages'
+
+    tags: []
```

actionlint (1.7.7)

"tags" section should not be empty

(syntax-check)